CHANGELOG.md

```diff
@@ -116,6 +116,7 @@ PLATFORM SPECIFIC
 
 ### windows
 	+ activated high precission timming to fix inaccurate fps calculations
+	+ allow passing -j<num_jobs> to compileOF.sh
```

Comment:
Ooops! Thanks for spotting, actually it's Linux only, my bad, early morning, first commit to oF ;)
Will fix, force-push and recreate pull request!